core/cu29_runtime/src/config.rs

```diff
@@ -705,12 +707,13 @@ mod tests {
     #[test]
     fn test_logging_parameters() {
         let txt =
-            r#"( tasks: [], cnx: [], logging: ( slab_size_mib: 1024, section_size_mib: 100, ),) "#;
+            r#"( tasks: [], cnx: [], logging: ( slab_size_mib: 1024, section_size_mib: 100, disable_logging: true ),) "#;
```

Comment:
I think "logging" can be confusing as the other parameters relates to the unified logger (slab and sections) while this one is about task logging.

Also, usually there is a rule where you don't want double negative in flags which confuses users.
what about "enable_task_logging" with default == True? to make the flag clearer?